tests/import-object.test.ts

```diff
@@ -1,10 +1,24 @@
 import { Type } from "../ast";
-import { NUM, BOOL, NONE, unhandledTag } from "../utils";
+import { NUM, STRING, BOOL, NONE, unhandledTag } from "../utils";
 
 function stringify(typ: Type, arg: any): string {
   switch (typ.tag) {
     case "number":
       return (arg as number).toString();
+    case "string":
+      if (arg == -1) throw new Error("String index out of bounds");
```

Comment:
Ah I see. @h9jiang FYI. I would make this a separate function rather than re-using the same one to report the error and print.

Reply:
@jpolitz We are unsure about this comment. Can you please clarify what the new function must be doing? For the case, arg == -1, should we call a new function where we just throw an error or should we wait for another group to implement this error handling?